report_xlsx_pdf/readme/INSTALL.rst

```diff
@@ -0,0 +1,10 @@
+Make sure you have ``xlsxwriter`` Python module installed::
```

Comment:
This comes from report_xlsx, so it is probably not needed here. The only thing needed is libreoffice in this case, as xlsxwriter comes from report_xlsx